src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    public void showError(String message) {
+        System.out.println(message);
+    }
+
+    public void showLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Returns the full command entered by the user.
+     * @return the next full line from the user input
+     */
+    public String readCommand() {
+        Scanner in = new Scanner(System.in);
+        return in.nextLine();
+    }
+
+    public void showLoadingError() {
+        System.err.println("☹ OOPS!!! I'm sorry, but I can't load your tasks :-(");
+    }
+
+    public void showSavingError() {
+        System.err.println("☹ OOPS!!! I'm sorry, but I can't save your tasks :-(");
+    }
+
+    public void showNoTasks() {
+        System.out.println("You have no tasks.\n" + "What can I do for you?");
+    }
+
+    /**
+     * Prints message after adding task with current TaskList size.
+     * @param task the task that was added
+     * @param size the size of the TaskList after adding the task
+     */
+    public void showAddTask(Task task, int size) {
```

Comment:
Just a small point for consideration, separating the operands to + in System.out.println() and aligning them neatly can give a better visualisation of what is meant to be printed in one quick glance.